C/CeresSolver/build_tarballs.jl

```diff
@@ -16,11 +16,12 @@ CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix}
               -DBUILD_SHARED_LIBS=ON
               -DBUILD_EXAMPLES=OFF
               -DBUILD_TESTING=OFF
-              -DBLAS_LIBRARIES=${libdir}/libopenblas.${dlext}
-              -DLAPACK_LIBRARIES=${libdir}/libopenblas.${dlext}
+              -DCMAKE_PROJECT_TOP_LEVEL_INCLUDES="${libdir}/cmake/blastrampoline/yggdrasilenv.cmake"
               -DMETIS_LIBRARY=${libdir}/libmetis.${dlext}
               )
 
+apk del cmake
+
 cd $WORKSPACE/srcdir/ceres-solver/
 mkdir build && cd build
 cmake .. ${CMAKE_FLAGS[@]}
```

Comment:
```suggestion
cmake -S .. -B . "${CMAKE_FLAGS[@]}"
```
Lets try this since it has a quoted string in it.